analyses/independent-samples/run-independent-samples.sh

```diff
@@ -3,15 +3,21 @@
 # Josh Shapiro for CCDL 2019
 #
 # Runs 01-generate-independent-specimens.R with default settings.
+# Takes one environment variable, `BASE_SUBTYPING`, if value is 1 then
+# uses pbta-histologies-base.tsv for subtyping if value is 0 runs all modules with pbta-histologies.tsv(Default)
 
 set -e
 set -o pipefail
 
+RUN_FOR_SUBTYPING=${BASE_SUBTYPING:-0}
+
 # Set the working directory to the directory of this file
 cd "$(dirname "${BASH_SOURCE[0]}")"
 
-Rscript -e "rmarkdown::render('00-repeated-samples.Rmd', clean = TRUE)"
+Rscript -e "rmarkdown::render('00-repeated-samples.Rmd',params=list(base_run = ${RUN_FOR_SUBTYPING}), clean = TRUE)"
 
+if [[ RUN_FOR_SUBTYPING == "0" ]]
```

Comment:
Same comment here about if statements. You can name a variable so you don't have to repeat whole chunks of code.